sql/2020/03_Markup/pages_element_count_by_device_and_custom_dash_elements.sql

```diff
@@ -0,0 +1,54 @@
+#standardSQL
+# % of pages having specific custom elements 
+# See related: sql/2019/03_Markup/03_03c.sql
+
+# 4.9 seconds for sample 10k
+# I tried using this to get the total in a shorter way. Could not get it to be accurate, and it was slower!:
+# COUNT(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
+
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+CREATE TEMPORARY FUNCTION get_element_types_with_a_dash(element_count_string STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS ''' 
+try {
+    if (!element_count_string) return ["ERROR: payload._element_count is missing"]; // 2019 had a few cases
```

Comment:
This is useful diagnostic info but not sure if we need it in the final results. Ideally the results are cleaned up for public consumption.

Reply:
Changed to return an empty array on any error or if nothing is detected.